build_scripts/windows/scripts/build.cmd

```diff
@@ -207,7 +207,7 @@ popd
 
 REM Remove __pycache__
 echo remove pycache
-for /d /r %BUILDING_DIR%\Lib\site-packages\pip %%d in (__pycache__) do (
+for /d /r %BUILDING_DIR%\Lib\site-packages\ %%d in (__pycache__) do (
```

Comment:
Pip pyc files are deleted in https://github.com/Azure/azure-cli/pull/6625.
Calling pip is slightly slower because it compiles byte code on the fly.